cypress/fixtures/users/credentials.json

```diff
@@ -0,0 +1,4 @@
+{
+  "email": "test@gmail.com",
+  "password": "Testing@123"
+}
```

Comment:
With the new login mocking, I think this file is safe to delete.

Reply:
@brentmclark Deleted this file